### DISCUSSION TITLE
Layer blending is broken

### DISCUSSION THREAD

**Discussion by wandrinsheep** [keiwando/toonsquid#482] [General]
Neither pixel layers nor animation layers blending is working at all or am I missing how to properly use this feature? It was working before not sure what’s causing the issue I’m experiencing

**Comment by keiwando** [comment]
I'm unable to reproduce any issues with layer blending. What exactly is happening if you change the blend mode or opacity of a layer? Can you maybe post a short screen-recording that shows the problem?

**Reply by wandrinsheep** [reply]
I found the cause. It’s just the layer had  clipping mask and the clipped layer doesn’t respond to blending changes of the layer it’s attached to

**Reply by wandrinsheep** [reply]
https://imgur.com/a/itroO7K 
‘I’ve gone ahead and done a small recording of the issue I’m facing. Basically clipping masks don’t inherit layer blending modes of the master layer.

**Reply by keiwando** [reply]
Thank you! I was able to reproduce the issue thanks to the video and will fix it with a future update. This bug currently only exists when masking and setting blend modes on animation layers. The equivalent drawing layer setup (pixel layer with overlay blend mode + masked pixel layer above) works correctly.
